src/PersistentLoggedStatement.php

```diff
@@ -164,7 +203,7 @@ public function errorCode() : ?string
         return $this->parent->errorCode();
     }
 
-    public function errorInfo() : ?array
+    public function errorInfo() : array
```

Comment:
BC break ... ?

Reply:
This relates to the comment I left in the description of the PR.

_Return type array|null of method Atlas\Pdo\PersistentLoggedStatement::errorInfo() is not covariant with tentative return type array of method PDOStatement::errorInfo()._

Either this change or `#[\ReturnTypeWillChange]`